docs/src/guide/installation.md

```diff
@@ -82,32 +82,30 @@ export PATH="/opt/doxypress:$PATH"
 
 #### ARM Cortex-M
 
-Install the [pre-built ARM toolchain][gcc-arm-toolchain]:
+Install the GNU toolchain for `arm-none-eabi` target in version 12 (or higher).
+If your Linux distribution provides up-to-date packages, we recommend using them.
```

Comment:
No? That's like the opposite of what we recommend 😜

Reply:
On linux systems with up-to-date packages (like Fedora or Arch) I recommend everyone to install the [distributions package(s)](https://packages.fedoraproject.org/pkgs/arm-none-eabi-gcc-cs/arm-none-eabi-gcc-cs-c++/) because it is so much less trouble (dependency management, some libs in wrong versions, $PATH, SElinux permissions, ...) than unpacking a tar archive to `/opt`.
(Only) the Ubuntu (22.04) package was broken in the past and is currently not up to date.

Reply:
Yeah ok, I see the problem. Let's just try it and see if anything breaks. I feel like arm-none-eabi-gcc is more popular now, and so I guess there are more people who care about it.